plugins/module_utils/repo_setup/main.py

```
@@ -357,6 +360,15 @@ def _validate_podified_ci_testing(repos):
                 "same time as other repos, except "
                 "deps|ceph."
             )
+    if "podified-ci-testing-tcib" in repos and len(repos) > 1:
+        if "deps" in repos or "ceph" in repos:
+            return True
+        else:
+            raise InvalidArguments(
+                "Cannot use podified-ci-testing-tcib at the "
+                "same time as other repos, except "
+                "deps|ceph."
+            )
     return True
```

I suggest to remove `return True` in L372 or change condition and leave the "return True":

```
    if "podified-ci-testing-tcib" in repos and len(repos) > 1:
        if "deps" not in repos or "ceph" not in repos:
            raise InvalidArguments(
                "Cannot use podified-ci-testing-tcib at the "
                "same time as other repos, except "
                "deps|ceph."
            )
```

---

So... this code is pretty-much copy paste of the existing one for podified-ci-testing and I am afraid of changing any logic here :pray:

---

is a not A and not B equal to that. But I'm fine maintaining in the same way of the other piece of code.

---

actually, it should be completely different condition, judging from the function's docstring and raised error ~ if there is anything in `repos` except `deps` or `ceph` it should raise an error; current code checks "oh, there is deps, so it must be fine" and does not check if there is something else – but I do not know here it may be used any relying on that wrong check, so that's why I am afraid of doing changes here :wink: